tests/Cluster/ClusterInfo-test.cpp

```diff
@@ -141,7 +141,8 @@ struct ClusterInfoSetup {
     arangodb::EngineSelectorFeature::ENGINE = &engine;
 
     // suppress INFO {authentication} Authentication is turned on (system only), authentication for unix sockets is turned on
-    arangodb::LogTopic::setLogLevel(arangodb::Logger::AUTHENTICATION.name(), arangodb::LogLevel::WARN);
+    // suppress WARNING {authentication} --server.jwt-secret is insecure. Use --server.jwt-secret-keyfile instead
+    arangodb::LogTopic::setLogLevel(arangodb::Logger::AUTHENTICATION.name(), arangodb::LogLevel::ERR);
```

Comment:
The superfluous error message issue should have been fixed by now.
Thus can we revert the change to the log level here?

Reply:
it's safe to ignore all errors from the authentication topic anyway, since there are tests to check invalid permissions
and there are quite a few test files with this change, hence I am not sure if it's worthwhile to revert

Reply:
Ok, fine. It's a change unrelated to the subject of this PR, but it doesn't do any harm here.